tools/wptserve/wptserve/server.py

```diff
@@ -342,6 +345,34 @@ def handle_connect(self, response):
             self.setup()
         return
 
+    def log_request(self, code="-", size="-"):
+        if isinstance(code, http.HTTPStatus):
+            code = code.value
+
+        self.logger.debug(
+            "{} - - [{}] {!r} {!s} {!s}".format(
```

Comment:
Could we use f-strings here?

Reply:
I tried this, but it turned it into a very long line.